src/ansible_navigator/actions/lint.py

```diff
@@ -0,0 +1,343 @@
+"""
+This action provides ansible-lint results through ansible-navigator.
+Internally, it works by using ansible-runner to execute ansible-lint (optionally
+in an execution environment). When running ansible-lint, it passes
+'-f codeclimate' which requests JSON output on stdout. The JSON output contains
```

Comment:
I think highlighting this would render more nicely:
```suggestion
``-f codeclimate`` which requests JSON output on stdout. The JSON output contains
```
See https://ansible-navigator--596.org.readthedocs.build/en/596/pkg/ansible_navigator.actions.lint/#module-ansible_navigator.actions.lint